go/logic/topology_recovery.go

```diff
@@ -250,6 +252,7 @@ func resolveRecovery(topologyRecovery *TopologyRecovery, successorInstance *inst
 	if successorInstance != nil {
 		topologyRecovery.SuccessorKey = &successorInstance.Key
 		topologyRecovery.SuccessorAlias = successorInstance.InstanceAlias
+		topologyRecovery.SuccessorBinlogCoordinates = fmt.Sprintf("%+v", successorInstance.SelfBinlogCoordinates)
```

Comment:
Assign actual coordinates

Reply:
Otherwise, only convert a `BinlogCoordinates` to string via `DisplayString()` and do not trust `'%+v` to be parseable.

Reply:
I would go with assigning actual coordinates